src/components/editor/InputLiteral.jsx

```diff
@@ -282,4 +260,4 @@ const mapDispatchtoProps = dispatch => ({
   }
 })
 
-export default connect(mapStatetoProps, mapDispatchtoProps)(InputLiteral);
+export default connect(null, mapDispatchToProps)(InputLiteral);
```

Comment:
i assume the null has actual meaning ...

Reply:
Required by the `connect` function if we don't have a `mapStateToProps` function.